contrib/utils.py

```diff
@@ -0,0 +1,59 @@
+import contextlib
+from typing import Generator
+from unittest.mock import patch
+
+
+@contextlib.contextmanager
+def production_endpoint() -> Generator:
```

Comment:
Is this really needed? With `HF_ENDPOINT` you could change the endpoint you're using

Reply:
> Is this really needed? 

I'd say yes, kinda the same as what already exists in the `hfh tests/`. The problem with `HF_ENDPOINT` is that it is evaluated only once at startup. What I want here is to make all calls to staging environment (especially pushing to repos) except some calls that have to be made to production environment (especially loading models).

Another solution could be to upload test models to staging but then we wouldn't notice if a model changed in production.